testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/cdi/injection/InjectionTest.java

```diff
@@ -276,7 +277,7 @@ public void testEJBs() throws Exception {
    public void testSessionScope() throws Exception {
       log.info("starting testSessionScope()");
       client.close();
-      client = ClientBuilder.newClient();
+      client = new ResteasyClientBuilderImpl().cookieManagementEnabled(true).build();
```

Comment:
Please do not reference the ResteasyClientBuilderImpl directly; use:

`client = ((ResteasyClientBuilder)(ClientBuilder.newBuilder())).cookieManagementEnabled(true).build();`

Reply:
Done